.changelog/2596.txt

```diff
@@ -1,3 +0,0 @@
-```release-note:enhancement
-Properly handle Kubernetes Jobs with ttl_seconds_after_finished = 0 to prevent unnecessary recreation.
-```
```

Comment:
Please don't remove this file. Instead, please add a new one that will reflect your changes. Your changes fall under the _bug fix_ category. Please mention there that this bug fix reverts changes introduced in 2.34.0.

Reply:
Makes sense, will do.